src/charm.py

```diff
@@ -1146,23 +1167,277 @@ def _addon_reconciliation(self, addon_type):
                 raise exceptions.WordPressBlockedStatusException(
                     f"failed to uninstall {addon_type} {repr(addon)}"
                 )
+
     def _theme_reconciliation(self):
         """Reconciliation process for WordPress themes
 
         Install and uninstall themes to match the themes setting in config
         """
         self._addon_reconciliation("theme")
 
+    def _wp_option_update(self, option, value, format="plaintext"):
+        """Create or update a WordPress option value
```

Comment:
(Applies to all docstrings) we have a new standard for docstrings which includes descriptions of parameters and return values: https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#docstrings

Reply:
Added, thanks.